dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -54,6 +54,9 @@ Future<void> main() async {
       final File marquee = File(path.join(flutterModuleLibSource.path, 'marquee'));
       marquee.copySync(path.join(flutterModuleLibDestination.path, 'marquee.dart'));
 
+      final File resize = File(path.join(flutterModuleLibSource.path, 'resize'));
+      resize.copySync(path.join(flutterModuleLibDestination.path, 'resize.dart'));
```

Comment:
nit: if you already have a `FileEntity` don't drop to a string path until you need it.
 
```suggestion
      resize.copySync(flutterModuleLibDestination.childFile('resize.dart').path);
```
This entire file does this, it doesn't really matter.